style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: 'Poppins';
+  src: url(fonts/Poppins-Regular.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Playfair Display';
+  src: url(fonts/PlayfairDisplay-Regular.ttf);
+  font-style: normal;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  color: #1a2236;
+}
+
+h2 {
+  font-family: 'Playfair Display', sans-serif;
+  font-size: 50px;
+  line-height: 100%;
+}
+
+h3 {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+}
+
+.content {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.logo-nav {
+  display: none;
+}
+
+.toggleNav {
+  display: none;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+nav a {
+  text-decoration: none;
+}
+
+nav li {
+  list-style: none;
+}
+
+.hero {
+  padding: 0 5vh 5vh 5vh;
+  margin: 0;
```

Comment:
- [ ] Please give the hero section a proper margin, referring to the original design.

Reply:
Well done!